### ISSUE TITLE
Future work

### ISSUE THREAD

**Issue by msavin** [cult-of-coders/redis-oplog#173]
Following up from the [pull request](https://github.com/cult-of-coders/redis-oplog/pull/172), I would like to make a few suggestions for marketing the package:

**Step 1: Give the package a new name**
  - The Redis Oplog as a name is great because its suggestive, but it may not be "technically correct". My suggestion would be to rebrand it in a way to highlight its uniqueness.
  - My suggestion for a name would be <a href="https://forums.meteor.com/t/meteor-scaling-redis-oplog-status-prod-ready/30855/262">Redtail</a> 
  - If agreed, I have an old logo that was in the works that I could have completed. We could dock a nice jet avatar similarly to this <a href="https://github.com/msavin/stevejobs">package</a>.

**Step 2: Create a new GitHub organization**
 - This will help if/when the package becomes a set of packages. It will also be more inviting for other people to join and contribute.
 - This could/should still be marked as Cult of Coders production - but the idea is to give the project its own legs

**Step 3: Create more documentation**
 - There is a lot that we can clarify up front, such as, what happens when you use Redtail with Kadira?

**Step 4: Create a simple website.**
  - Having markdown documents in a repo is nice, but we can really improve the presentation by building a separate website. 
  - Perhaps we could start with something simple hosted on GitHub pages

Once 1 and 2 are set, I could help with step 3 and 4. After that, we could evaluate what v2 might look like, what the licensing terms would be like, and so on.

**Comment by ramezrafla** [comment]
How about **Scalable LiveQuery** as a name? Redtail doesn't speak in itself, we need to read further to get it. Or **RedisTail** if you really want Redis in there. I like Scalabke LiveQuery as it speaks to what it does more than how it does it

**Comment by msavin** [comment]
I wouldn't call it Scalable LiveQuery because it sounds like we are trying to one-up MDG in that case. However, the subtitle could be, "A Redis-based Scaling Solution for LiveQuery"

I think Redtail is nicer because its short, distinct, and we can use the redtail airplane as a logo. It would play nicely into the theme of the package: Redtail makes your app fly.

For some reason,  I think of Bower in this context - maybe because birds also fly. I really like how nice and simple the [website](https://bower.io/) for it.

**Comment by msavin** [comment]
Lol - just for fun: 

<img width="1280" alt="screen shot 2017-10-08 at 5 36 31 pm" src="https://user-images.githubusercontent.com/2031235/31318237-48997a32-ac4f-11e7-81fe-1f6a84808b37.png">

**Comment by ramezrafla** [comment]
We call this coercion :) well done 
Looks great!

**Comment by msavin** [comment]
Hmmm - is there a word for something in between persuasion and passion? 😁

**Comment by ramezrafla** [comment]
Is the name taken? http://corporate.redtailtechnology.com/

**Comment by msavin** [comment]
I don't think it would be a problem because we are in totally different markets - but we'd need to  ask a lawyer to know for sure.
